src/view.tree.js

```diff
@@ -23,15 +23,17 @@ class TreeView {
     $jstree.settings.core.data = (node, cb) => {
       const loadAll = this.adapter.canLoadEntireTree() &&
                       this.store.get(STORE.LOADALL)
-      node = !loadAll && (node.id === '#' ? {path: ''} : node.original)
+      // No lazy loading in PR tree view
+      const isPR = this.store.get(STORE.PR) && repo.pullNumber
```

Comment:
This seems to leak GitHub specific logic into the UI.  I suggest encapsulate it into `adapter#canLoadEntireTree` if possible.

Reply:
Actually, that's a bad idea :D. Let me think more about this.

Reply:
This is needed because we need to call `this._collapse` down on line 34 to collapse nested empty folders. Without the added `|| isPR` check, the tree looks like this:
<img width="340" alt="screen shot 2017-08-31 at 2 27 13 pm" src="https://user-images.githubusercontent.com/3138607/29946048-97effcc4-8e58-11e7-97ce-0cebb24ed496.png">

Reply:
I got that part, I was asking about line 28 `node = ...`.